mula/scheduler/server/serializers/task.py

```diff
@@ -37,8 +36,18 @@ class Task(BaseModel):
 
     scheduler_id: str | None = None
 
+    schedule_id: uuid.UUID | None = None
+
+    priority: int | None = 0
```

Comment:
The `None` doesn't make sense if it's not nullable (I inferred this from the `0` assignment)

Reply:
This has to do with these models being added specifically to allow partial updates from the API patch endpoints. However, it is one of the issues that I want to tackle in the fastapi ticket we have for the scheduler. For now I'll set the assignment to `None`